packages/docs/package.json

```diff
@@ -56,6 +56,7 @@
     "gl-matrix": "^3.4.3",
     "hammerjs": "^2.0.8",
     "prism-react-renderer": "2.4.0",
+    "patch-package": "8.0.0",
```

Comment:
not needed anymore

Reply:
I'm just trying things out, not sure why our docs not building anymore